tests/Monolog/Handler/ChromePHPHandlerTest.php

```diff
@@ -47,6 +54,16 @@ public function testHeaders()
         $this->assertEquals($expected, $handler->getHeaders());
     }
 
+    public function agentsProvider()
```

Comment:
I suggest using a static method, which avoids PHPUnit to instantiate the TestCase when running the provider during the loading of the testsuite